sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -475,3 +474,26 @@ export interface RestoreKeyBackupOptions extends coreHttp.OperationOptions {}
  * An interface representing the options of the cryptography API methods, go to the {@link CryptographyClient} for more information.
  */
 export interface CryptographyOptions extends coreHttp.OperationOptions {}
+
+/**
+ * Options for {@link KeyClient.getRandomBytes}
+ */
+export interface GetRandomBytesOptions extends coreHttp.OperationOptions {}
+
+/** Known values of {@link KeyOperation} that the service accepts. */
+export enum KnownKeyOperations {
```

Comment:
What's this for?

Reply:
This doesn’t change the public API, it’s a copy of a type provided in the generated code.

Reply:
To expand on what @sadasant said - this is how TS supports extensible enums. 

While the type itself is just an alias to a string (like `export type KeyOperation = string` ), these can be used interchangeably with that type since they are strings.

Anyway, the reason I copied them over is we were re-exporting the generated code directly and I did not want to expose `Export` yet even in beta (which got added when I regenerated from 7.3)